vivisect/__init__.py

```diff
@@ -1672,7 +1673,7 @@ def getFunctionBlocks(self, funcva):
             ret = []
         return ret
 
-    def makeFunctionThunk(self, fva, thname, addVa=True, filelocal=False):
+    def makeFunctionThunk(self, fva, thname, addVa=True, filelocal=False, basename=None):
```

Comment:
`makeFunctionThunk` can now create `plt_*` entries but still maintain the real THUNK (*.foo) for Posix/ELF.

Reply:
This function is kinda getting complicated. From a consumer perspective, what's the difference between `thname` and `basename`? The way it's used down below doesn't really give a good indication of why `basename` is called basename. I think we should talk this one out a bit.

Reply:
added pertinent docstr info.  hopefully it clarifies this.

Reply:
:+1: